### ISSUE TITLE
Not being awarded COPR badges

### ISSUE THREAD

**Issue by mymindstorm** [fedora-infra/tahrir#390]
Datagrepper: https://apps.fedoraproject.org/datagrepper/raw?user=mymindstorm&topic=org.fedoraproject.prod.copr.build.end
Badges: https://badges.fedoraproject.org/user/mymindstorm

I got it to grant it to me, but only after I manually started a build. Looks like automated builds don't trigger the check for it.

**Comment by justwheel** [comment]
@mymindstorm I think this is best filed as a new bug on [pagure.io/fedora-badges](https://pagure.io/fedora-badges/). That repository contains the YAML rule files with the fedmsg queries to award and trigger badges (here is a [COPR badge rule](https://pagure.io/fedora-badges/blob/master/f/rules/copr-build-04.yml)).

Could you open this as an issue on that repository?

**Comment by mymindstorm** [comment]
Closing in favor of https://pagure.io/fedora-badges/issue/668

**Comment by justwheel** [comment]
@mymindstorm Thanks! :+1: Will follow up there.
